Examples/Scripts/Python/MLAmbiguityResolution/seed_solver_network.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.utils
+
+import ast
+
+
+def prepareDataSet(data: pd.DataFrame) -> pd.DataFrame:
+    """Format the dataset that have been written from the Csv file"""
+    """
+    @param[in] data: input DataFrame containing 1 event
+    @return: Formatted DataFrame 
+    """
+    data = data
```

Comment:
```suggestion
```

Reply:
I thing there is a reason for that one. I remember having to write this or it wouldn't work... Let me see if I can recall why

Reply:
Well removing seems to be ok so lets do it